src/unity/schemas/Organizations.yml

```diff
@@ -1,3 +1,13 @@
 type: array
 items:
-  $ref: './Organization.yml'
+  allOf:
+  - $ref: './Organization.yml'
+  - properties:
```

Comment:
operator get organizations returns a list of orgs with nested resources

- org
  - account
    - billing contact